cli/commands/help/index.md

```diff
@@ -1,21 +1,28 @@
 ---
-title: Help
+title: Help Command
 permalink: /cli/commands/help/
 ---
 
 * TOC
 {:toc}
 
+**Shows help for any command.**
+
 ```
 kbc help [command] [flags]
 ```
 
-Shows help for any command. Type `kbc help [command]` for full details. 
-
 ## Options
 
 [Global Options](/cli/commands/#global-options)
 
+## Example
+
+```
+kbc help local create config
+```
+
 ## Next Steps
 
+- [All Commands](/cli/commands/)
 - [Init](/cli/commands/init/)
```

Comment:
Tenhle link nefunguje, má tam být `/cli/commands/sync/init/`

Reply:
Taky z `http://localhost:4000/cli/commands/local/create/config/` link na `row`
a z `http://localhost:4000/cli/commands/local/create/row/` na `diff`

Reply:
- z `http://localhost:4000/cli/commands/local/persist/` na `diff`
- z `http://localhost:4000/cli/commands/local/encrypt/` na `fix-paths`
- z `http://localhost:4000/cli/commands/local/validate/` na `workflows`
- z `http://localhost:4000/cli/commands/remote/create/branch/` na `create-config`
A to je myslím všechno.

Reply:
Dik, fixoval som to, no zjavne som niektore miesta preskocil :beers: